MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.h

```diff
@@ -47,7 +47,7 @@ typedef NS_ENUM(NSUInteger, MUXSDKViewOrientation) {
     AVPlayer *_player;
     AVPlayerItem *_playerItem;
     id _timeObserver;
-    MUXSDKPlayerState _state;
+    volatile MUXSDKPlayerState _state;
```

Comment:
wasn't this already included in a previous PR?

Reply:
yeah I need to update this PR with the lastest release. I opened it before merging those.